doc/source/guidelines/index.rst

```diff
@@ -0,0 +1,20 @@
+Guidelines
+#############################
+The purpose of this section is to highligh some common practices that
+can be applied to the entire PyAnsys project in order to remain consistent.
+
+One of the main objectives of PyAnsys libraries is to wrap (encapsulate) 
+data and methods within units of execution while hiding data or parameters 
+in protected variables.
```

Comment:
@akaszynski I kept this paragraph because it is quite generic. But I am not really sure it fits here.

Reply:
I think it's fine to leave this in.